examples/data-objects/codemirror/src/index.ts

```diff
@@ -63,22 +63,23 @@ class CodeMirrorFactory extends RuntimeFactoryHelper {
 		context: IContainerContext,
 		existing: boolean,
 	): Promise<ContainerRuntime> {
-		const registry = new Map<string, Promise<IFluidDataStoreFactory>>([
+		const registryEntries = new Map<string, Promise<IFluidDataStoreFactory>>([
 			[smde.type, Promise.resolve(smde)],
 		]);
 
-		const runtime: ContainerRuntime = await ContainerRuntime.load(
+		const runtime: ContainerRuntime = await ContainerRuntime.loadRuntime({
 			context,
-			registry,
+			registryEntries,
 			// eslint-disable-next-line import/no-deprecated
-			buildRuntimeRequestHandler(
+			requestHandler: buildRuntimeRequestHandler(
 				// eslint-disable-next-line import/no-deprecated
 				mountableViewRequestHandler(MountableView, [viewRequestHandler]),
 			),
-			undefined, // runtimeOptions
-			undefined, // containerScope
+			provideEntryPoint: () => {
+				throw new Error("TODO: AB#4990");
```

Comment:
this is an odd sequencing of changes to me. i think it would be better to first remove all usages, and then remove the methods. i don't like that addition of all the todos.